### PR TITLE
Add structural ID mapping and setup for `CAVEclient`

### PR DESCRIPTION
**Pull request by weiglszonja**
The token for `CAVEclient` should be stored in an environment variable named CAVE, this is only needed the first time the client is accessed.

### CODE REVIEW

File: src/microns_to_nwb/tools/ophys/ophys.py [hunk 1]
```diff
@@ -75,6 +79,68 @@ def add_plane_segmentation(field_key, nwb, imaging_plane, image_segmentation):
...
+        description="The location in 4,4,40 nm voxels at a cell body for the cell.",
+        data=pt_supervoxel_ids,
+    )
+
+
+def get_functional_coregistration(field_key):
+    try:
+        token = get_token_from_external_file("../../caveclient_token.json")
+        client = get_client(token=token)
+    except Exception as e:
```
Comment (CodyCBakerPhD) [inline src/microns_to_nwb/tools/ophys/ophys.py]: What kind of errors did you see?

**Comment by CodyCBakerPhD** [comment]
Cool, is there an example output of this plane segmentation table up on DANDI yet?

**Comment by CodyCBakerPhD** [comment]
Access scripts originally provided by Paul are no longer working, Szonja is asking about it now

File: src/microns_to_nwb/tools/cave_client/cave.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    with open(external_file_path) as f:
+        external_file = json.load(f)
+        assert "token" in external_file, f"'token' field is missing from {external_file_path}."
+
+    return external_file["token"]
+
+
+def get_functional_coreg_table(scan_key, token_file_path):
+    token = get_token_from_external_file(token_file_path)
+    client = get_client(token=token)
```
Comment (bendichter) [inline src/microns_to_nwb/tools/cave_client/cave.py]: is this the only place the token is used?

Comment (weiglszonja) [reply src/microns_to_nwb/tools/cave_client/cave.py]: yes I think so

Comment (weiglszonja) [reply src/microns_to_nwb/tools/cave_client/cave.py]: todo: set environment variable with token, read from env var

Comment (CodyCBakerPhD) [reply src/microns_to_nwb/tools/cave_client/cave.py]: This is pretty easy to do via `os`

```python
import os

os.environ["NAME_OF_ENVIRONMENT_VARIABLE"]
```

(note if using ipython, it must be set in the ipykernel prior to spawning the instance you might be running code in - and different virtual/conda environments do not share global environment variables by default)

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**